agent_workspace/src/App.css

```
@@ -0,0 +1,347 @@
+  --gradient-2: linear-gradient(135deg, #0a1628 0%, #112240 100%);
+  --glow-shadow: 0 0 20px rgba(0, 255, 136, 0.3);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Orbitron', 'Segoe UI', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)